### PR TITLE
T2D-269 update metadata dev url and point prod and fallback url to same db

### PR DESCRIPTION
**Pull request by selvaebi**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ pipeline {
   }
   environment {
     stagingPostgresDbUrl = credentials('STAGINGMETADATAWSDBURL')
-    fallBackPostgresDbUrl = credentials('FALLBACKMETADATAWSDBURL')
+    fallBackPostgresDbUrl = credentials('PRODMETADATAWSDBURL')
```
Comment (Zhicheng-Liu) [inline Jenkinsfile]: I think we need a discussion on how we want to deploy the DBs.

File: Jenkinsfile [hunk 2]
```diff
@@ -55,7 +55,7 @@ pipeline {
       steps {
         echo 'Deploying to Staging'
         sh "curl --upload-file staging/target/metadata-ws-*.war \
-        'http://'${tomcatCredentials}'@'${stagingHost}':8080/manager/text/deploy?path=/metadata&update=true' | \
+        'http://'${tomcatCredentials}'@'${stagingHost}':8080/manager/text/deploy?path=/dev/metadata&update=true' | \
```
Comment (Zhicheng-Liu) [inline Jenkinsfile]: Shouldn't we in proxy settings map `/dev/metadata` to `http://staging:8080/metadata`?

Comment (selvaebi) [reply Jenkinsfile]: the Swagger base path takes host + prefix + servletContext only. So the if we deploy using /metadata - the staging app base url forms to ebi.ac.uk+ ega/ampt2d + metadata - which is again prod app base url.

Comment (Zhicheng-Liu) [reply Jenkinsfile]: Ok! Then I am fine with this change.

File: Jenkinsfile (hunk 1 above)
Comment (selvaebi) [reply Jenkinsfile]: So until we get a production postgres and proxy corrected we gonna use same database for fallback and prod

Comment (Zhicheng-Liu) [reply Jenkinsfile]: OK! Once we have production instances set up and properly configured for cross instance replication we can use different databases for fallback and production. For now, we have to use the same database.

File: metadata-ws/Jenkinsfile [hunk 3]
```diff
@@ -6,7 +6,7 @@ pipeline {
   }
   environment {
     stagingPostgresDbUrl = credentials('STAGINGMETADATAWSDBURL')
-    fallBackPostgresDbUrl = credentials('FALLBACKMETADATAWSDBURL')
+    fallBackPostgresDbUrl = credentials('PRODMETADATAWSDBURL')
```
Comment (iyangar) [inline metadata-ws/Jenkinsfile]: Is this because we don't have dedicated server at this moment.

**Pull request merged**